tests/index.rst

```diff
@@ -29,6 +29,7 @@ Welcome to sphinxcontrib-jupyter.minimal's documentation!
    only
    quote
    simple_notebook
+   Slideshow
```

Comment:
I know its pedantic `:-)` but can you please use lowercase `slideshow` for consistency.